src/viewer.js

```diff
@@ -2132,6 +2132,58 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      */
     _cancelPendingImages: function() {
         this._loadQueue = [];
+    },
+
+    /**
+     * Remove the reference strip.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    removeReferenceStrip: function() {
```

Comment:
Good point! I think it's appropriate for removeReferenceStrip to set showReferenceStrip to false. 

I see you found a solution by adding the `hide` parameter, but I think that complicates the API. Let's not call removeReferenceStrip from `open` after all instead; sorry for the back and forth on this!